sceptre-example/config/special-file-asset-cdk-json.yaml

```diff
@@ -0,0 +1,31 @@
+# This template is an example of using the CDK template handler to deploy with a stack from a CDK
+# project that is not developed in Python. In this example, we're demonstrating how a TypeScript
+# stack Construct can be deployed via Sceptre using this handler.
+
+template:
+  type: cdk
+  # The only real difference is that the path points to the `cdk.json` file at the root of a CDK
+  # project instead of a python file. The CDK Handler is smart enough to synthesize the template and
+  # publish any assets using the CDK CLI. This lets the handler support all CDK-supported languages
+  # rather than only Python.
+  path: ../cdk-app/cdk.json
```

Comment:
Yes, it's because the cdk.json is _outside_ the templates directory. Since a CDK project can be a lot more substantial than a single template file, it's likely users of this handler would want to place their CDK project outside of `templates`. In several senses, a CDK project is not a "template" and it doesn't make so much sense to bury a CDK project within a directory named `templates`. In fact, with the way that CDK initializes repositories and such, it generally makes more sense to have the CDK project be in a different directory than the Sceptre stuff. 

I could put a comment to this effect here to clear up the confusion, though.

Reply:
yes, users would probably be confused without a little explanation.